### PR TITLE
BBM Profile Box docs

### PR DESCRIPTION
**Pull request by ericpearson**

### CODE REVIEW

File: api/blackberry_bbm_platform_self_profilebox_ProfileBoxItem.js [hunk 1]
```diff
@@ -22,9 +22,18 @@
...
+     * @readOnly
+     * @BB10X
+     */
+
+    /**
+     * The item ID.
      * @type Number
      * @readOnly
      * @BB50+
+     * @BB10X
```
Comment (jeffheifetz) [inline api/blackberry_bbm_platform_self_profilebox_ProfileBoxItem.js]: Wait didn't you just say above that the id was a String?

Comment (ericpearson) [inline api/blackberry_bbm_platform_self_profilebox_ProfileBoxItem.js]: I will fix it thanks @jeffheifetz

**Comment by jeffheifetz** [comment]
I don't see all the functions displayed in either our CI or the github pages build

**Comment by ericpearson** [comment]
I don't really know why it's not showing up, if you generate the docs locally you can see everything

**Comment by jeffheifetz** [comment]
Merged Manually

**Pull request closed without merging**